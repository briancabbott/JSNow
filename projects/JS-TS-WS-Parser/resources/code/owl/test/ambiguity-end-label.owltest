---
a = b b
b = c?
c = 'y'*
