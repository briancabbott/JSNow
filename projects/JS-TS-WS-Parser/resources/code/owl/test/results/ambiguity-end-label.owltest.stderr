error: this grammar is ambiguous

. 

  can be parsed in two different ways: as

.    
  b b
  a--

  or as

.    
  | c
  b b
  a--

